src/index.css

```diff
@@ -1,27 +1,7 @@
-body {
+* {
   margin: 0;
-  font-family:
-    -apple-system,
-    BlinkMacSystemFont,
-    'Segoe UI',
-    'Roboto',
-    'Oxygen',
-    'Ubuntu',
-    'Cantarell',
-    'Fira Sans',
-    'Droid Sans',
-    'Helvetica Neue',
-    sans-serif;
-  -webkit-font-smoothing: antialiased;
-  -moz-osx-font-smoothing: grayscale;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
-}
+/* body {
+    background-color: red;
+} */
```

Comment:
- [OPTIONAL] It's against the best practices to leave unused code as a comment in your codebase, I'll suggest you remove this. :+1: :100: :smile:

Reply:
thanks for noticing, I'm on it!